api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -804,13 +810,11 @@ public DbUser confirmPublications() {
     return accessSyncService.updateUserAccessTiers(dbUser, Agent.asUser(dbUser));
   }
 
-  /** Send an Access Renewal Expiration or Warning email to the user, if appropriate */
+  /** Send Access Renewal Expiration or Warning emails to the user for all tiers, if appropriate */
```

Comment:
This comment increases the vertical space consumed by 25%. Consider whether or not it is adding enough value to pay for that space.